{{cookiecutter.project_dashed}}-src/{{cookiecutter.project_namespace}}/tests/test_auth.py

```diff
@@ -5,4 +5,4 @@
 
 class TestAuthIntegration(AuthTests):
     user_ent = ents.User
-    protected_url = '/protected-example'
+    protected_url = '/users'
```

Comment:
Tests fail because of this change. AuthTests assume that any logged-in user can access the `protected_url`, but the users route requires an extra permission. `auth-manage` I believe.

Reply:
The `AuthTests` have been changed to fix this, ci is now passing